wren-ui/src/pages/home/index.tsx

```diff
@@ -119,18 +114,13 @@ export default function Home() {
   );
 
   const onSelectQuestion = async ({ question }) => {
-    $prompt.current.setValue(question);
-    await nextTick();
-    $prompt.current.submit();
+    $prompt.current.submit(question);
   };
 
-  const onSelect = async (payload: CreateThreadInput) => {
+  const onCreateResponse = async (payload: CreateThreadInput) => {
     try {
       askPrompt.onStopPolling();
       const response = await createThread({ variables: { data: payload } });
-      await generateThreadRecommendationQuestions({
```

Comment:
remove useGenerateThreadRecommendationQuestionsMutation since it is no longer used in here.